src/olympia/migrations/1005-convert-average-rating-to-float.sql

```diff
@@ -0,0 +1,5 @@
+-- Before converting to a double, get rid of the empty string values we can't
+-- convert.
+UPDATE users SET averagerating=NULL WHERE averagerating = "";
+-- Make it a double.
+ALTER TABLE users MODIFY averagerating double;
```

Comment:
I'm actually kind of impressed mysql/mariadb can do that conversion without blowing up

Reply:
Yeah, me too. It does blow up on empty strings, but works fine with `NULL` and floats-as-strings.